neo/Ledger/MemoryPool.cs

```diff
@@ -192,10 +180,8 @@ public IEnumerable<Transaction> GetVerifiedTransactions()
             _txRwLock.EnterReadLock();
             try
             {
-                verifiedTransactions = _sortedHighPrioTransactions.Reverse().Select(p => p.Tx)
-                    .Concat(_sortedLowPrioTransactions.Reverse().Select(p => p.Tx)).ToArray();
-                unverifiedTransactions = _unverifiedSortedHighPriorityTransactions.Reverse().Select(p => p.Tx)
-                    .Concat(_unverifiedSortedLowPriorityTransactions.Reverse().Select(p => p.Tx)).ToArray();
+                verifiedTransactions = _sortedTransactions.Reverse().Select(p => p.Tx).ToArray();
+                unverifiedTransactions = _unverifiedSortedTransactions.Reverse().Select(p => p.Tx).ToArray();
```

Comment:
Good point...

Reply:
You need to reverify the fees.

Reply:
We need a pre-balance structure to make this verification a trivial thing.

Reply:
I marked this is unresolved because I believe this should be clear when this PR is finished, the direction we are going to take for this verifications.

Reply:
We should improve the re-verification in another PR.